lua/cheatsheet/telescope.lua

```diff
@@ -80,9 +81,37 @@ M.pick_cheat = function(opts)
             attach_mappings = function(prompt_bufnr, map)
                 actions.select_default:replace(
                     function()
-                        -- local selection = action_state.get_selected_entry()
-                        -- close telescope on Enter
+                        local selection = actions_state.get_selected_entry()
+                        local cheat = selection.value.cheatcode
+                        local description = selection.value.description
+
                         actions.close(prompt_bufnr)
+
+                        if string.len(cheat) > 1 then
+                            -- check for valid command
+
+                            cheat = cheat:match("^%s*(.-)%s*$") -- strip spaces
```

Comment:
Spaces are already stripped when parsing the cheatsheet :)

https://github.com/sudormrfbin/cheatsheet.nvim/blob/cf5ef0899b595d7c16952184a9be2c69592d2e54/lua/cheatsheet/init.lua#L27